### PR TITLE
Prepare HISTORY file and add auto-generated CHANGELOG file

### PR DESCRIPTION
**Pull request by christianhueserhzdr**
* Prepare HISTORY.md with previous CHANGELOG and exchange links to point
to GitHub.
* Auto-generate CHANGELOG.md which includes HISTORY entries.
* Rename GITLAB_BASE_CHANGELOG to GITLAB_BASE_HISTORY.

### CODE REVIEW

**Comment by christianhueserhzdr** [comment]
Dear Norman @Normo, I tried generating the CHANGELOG file. Could you please check that everything is alright? Thanks.

File: HISTORY.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# Changelog
+
+## [Unreleased](https://gitlab.com/hifis/ansible/gitlab-role/tree/HEAD)
+
+[List of commits](https://gitlab.com/hifis/ansible/gitlab-role/-/compare/v1.1.0...main)
+
+### Changed
```
Comment (Normo) [inline HISTORY.md:14]:
```suggestion
**Changed**
```

Comment (christianhueserhzdr) [reply HISTORY.md]: Dear Norman @Normo, thank you for your suggestion. I changed the HISTORY file and re-generated the CHANGELOG file. Is this fine from your point of view? Thanks for checking again.

**Review by Normo — APPROVED** [review]
LGTM :+1:  Thanks, @christianhueserhzdr.

**Pull request merged**